openslides_backend/shared/exceptions.py

```diff
@@ -23,6 +23,10 @@ class ActionException(View400Exception):
     pass
 
 
+class NoContentException(BackendBaseException):
```

Comment:
I would suggest to subclass from `ViewException` and set the status code to 204. The rest should be handled automatically then and you can remove the try...catch block

Reply:
Or just throw away this exception altogether, see https://github.com/OpenSlides/openslides-backend/pull/245#discussion_r517444090

Reply:
I would also prefer not to use an exception here (although I might have wrote something else some days ago).